packages/expo-dev-launcher/ios/EXDevLauncherURLHelper.swift

```diff
@@ -12,7 +12,9 @@ public class EXDevLauncherURLHelper: NSObject {
   @objc
   public static func replaceEXPScheme(_ url: URL, to scheme: String) -> URL {
     var components = URLComponents.init(url: url, resolvingAgainstBaseURL: false)!
-    components.scheme = scheme
+    if (components.scheme == "exp") {
+      components.scheme = scheme
+    }
```

Comment:
in this case I think the failing test actually caught a bug -- I've updated this logic to match Android, @lukmccall does this look right to you?

Reply:
Yes, you're right